src/main/resources/public/ts/utilities/booking.ts

```diff
@@ -0,0 +1,22 @@
+import {_, Behaviours} from "entcore";
+
+export class BookingUtil {
+
+    /**
+     * Method that update/set `myRights` resource if data booking is not updated properly
+     * @param booking Booking
+     */
+    static setRightsResourceBooking(booking: any): void {
+        if ('myRights' in booking && _.isEmpty(booking.myRights)) {
```

Comment:
booking.myRights && booking.myRights.length > 0

Reply:
cannot read property of undefined

Reply:
booking && booking.myRights && booking.myRights.length > 0

Reply:
je suis d'accord pour la vérif mais par pour la 3eme